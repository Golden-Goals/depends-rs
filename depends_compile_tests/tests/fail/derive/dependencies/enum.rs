use depends::derives::Dependencies;

#[derive(Dependencies)]
enum Bar {
    Node1(usize),
    Node2(usize),
}

fn main() {}
