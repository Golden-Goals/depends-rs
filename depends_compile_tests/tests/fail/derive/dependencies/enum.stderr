error: expected `struct`
 --> tests/fail/derive/dependencies/enum.rs:4:1
  |
4 | enum Bar {
  | ^^^^
